spack/ci-spack-pyprecice-deps-1804.dockerfile

```diff
@@ -0,0 +1,8 @@
+# Build stage with Spack pre-installed and ready to be used
+FROM spack/ubuntu-bionic:latest
+RUN spack env create precice
+# Install dependencies for precice and py-pyprecice
+RUN spack env activate precice \ 
+    && spack add boost@1.74.0 cmake@3.18.4 eigen@3.3.8 petsc@3.14.1 pkgconfig precice@develop py-setuptools py-wheel py-cython py-numpy py-mpi4py \
+    && spack concretize -f \
+    && spack install
```

Comment:
Is this acceptable with regards to compile time or do we need to create a Docker image that already contains all the dependencies?

Reply:
The image is compiled locally (once) an provided on dockerhub (https://hub.docker.com/r/benjaminrueth/ci-spack-pyprecice-deps-1804). This Dockerfile is only provided in case we want to rebuild the image.

Reply:
That sounds good, too. I was not aware of the actual workflow.